src/js/node/dns.js

```diff
@@ -683,6 +684,28 @@ for (const key of ["resolveAny"]) {
   promises[key] = () => Promise.resolve(undefined);
 }
 
+// Compatibility with util.promisify(dns[method])
+for (const [method, pMethod] of [
+  [lookup,        promises.lookup],
+  [lookupService, promises.lookupService],
+  [resolve,       promises.resolve],
+  [reverse,       promises.reverse],
+  [resolve4,      promises.resolve4],
+  [resolve6,      promises.resolve6],
+  [resolveAny,    promises.resolveAny],
+  [resolveCname,  promises.resolveCname],
+  [resolveCaa,    promises.resolveCaa],
+  [resolveMx,     promises.resolveMx],
+  [resolveNs,     promises.resolveNs],
+  [resolvePtr,    promises.resolvePtr],
+  [resolveSoa,    promises.resolveSoa],
+  [resolveSrv,    promises.resolveSrv],
+  [resolveTxt,    promises.resolveTxt],
+  [resolveNaptr,  promises.resolveNaptr],
+]) {
+  method[util.promisify.custom] = pMethod
```

Comment:
```suggestion
  method[utilPromisifyCustomSymbol] = pMethod
```

Reply:
I'm still trying to find out how to force bun to reload `bun/src/js/node/*` assets and then apply to tests.

Reply:
Finally, __it works on my machine__